src/main/java/com/exasol/errorreporting/ErrorMessageBuilder.java

```diff
@@ -23,54 +20,53 @@
         this.errorCode = errorCode;
     }
 
-    /**
-     * Add exception message.
-     * <p>
-     * If this method is called multiple times, the message is appended.
-     * </p>
-     *
-     * @param message exception message
-     * @return self for fluent programming
-     */
-    public ErrorMessageBuilder message(final String message) {
-        this.messageBuilder.append(message);
-        return this;
-    }
-
     /**
      * Format a given message pattern with place holders, filling them with the arguments passed in the specified form.
      *
-     * Place holders are defined in the message pattern by using curly brackets `{}`. By default, arguments are
-     * formatted with simple quotes unless specified other wise with the 'unquoted' format, defined by `{|uq}`.
+     * Place holders are defined in the message pattern by using double curly brackets `{{}}`. By default, arguments are
```

Comment:
Even so it is currently possible to have a placeholder without a name, I would prefer to document here that users should use a name since the crawler will enforce it,